2020/CG-02.md

```diff
@@ -51,37 +51,40 @@ Registration is now closed, email [WebAssembly CG chair](mailto:webassembly-cg-c
 ## Agenda items
 
  - Day 1:
-   - [Bulk Memory Operations](https://github.com/WebAssembly/bulk-memory-operations)
+   - [Bulk Memory Operations](https://github.com/WebAssembly/bulk-memory-operations) (Ben Smith)
      - Poll to Phase 4? 
-   - [Reference Types](https://github.com/WebAssembly/reference-types)
+   - [Exception Handling](https://github.com/WebAssembly/exception-handling) (Heejin Ahn)
+     - Poll to Phase 2
+   - Stack switching / Coroutines / Effect handlers (Andreas Rossberg)
+   - [Reference Types](https://github.com/WebAssembly/reference-types) (Andreas Rossberg)
      - https://github.com/WebAssembly/reference-types/issues/76
      - Poll to Phase 4?
-   - Stack switching / Coroutines / Effect handlers
-   - [Exception Handling](https://github.com/WebAssembly/exception-handling) 
-     - Poll to Phase 2
-   - [Reword Web VM requirement](https://github.com/WebAssembly/meetings/issues/350)
-   - [Garbage Collection](https://github.com/WebAssembly/gc)
+   - [Reword Web VM requirement](https://github.com/WebAssembly/meetings/issues/350) (Ben Smith?)
+   - [Garbage Collection](https://github.com/WebAssembly/gc) (Andreas Rossberg)
    - [Debugging Topics](https://github.com/WebAssembly/debugging/issues)
      - Limitations of Source maps
-   - [Multi-value](https://github.com/WebAssembly/multi-value)
+   - [Multi-value](https://github.com/WebAssembly/multi-value) (Andreas Rossberg)
```

Comment:
A note here that @tlively will be presenting the toolchain update based on recently landed support, and it's currently slotted for ~15 Mins. If there's more to discuss we may have to bump it to the next day.

Reply:
Sounds good, added Thomas.